docs/multi_sources.rst

```diff
@@ -0,0 +1,32 @@
+========================
+Including multiple Sources
+========================
+
+In a rule you can configure match one or more source, each emitting events in different format. 
+
+Example::
+
+    sources:
+      - range:
+          limit: 6
+      - range2:
+          limit: 5
+
+The condition can match events from either source::
+
+    rules:
+      - name:
+        condition: event.i == 2
+        action:
+          debug:
+      - name:
+        condition: event.range2.i == 1
+        action:
+          debug:
+
+To avoid name conflicts the source data structure can use nested keys. The above examples assumes ``range`` 
```

Comment:
@bzwei @benthomasson 
Shouldn't it be mandatory for the plugins to send the events with the name of the plugin for clarity and to avoid possible collisions?